### PR TITLE
Add secfixes comments

### PR DESCRIPTION
**Pull request by J0WI**
Please backport the security fixes in `main/file` to stable branches.

### CODE REVIEW

File: main/file/APKBUILD [hunk 1]
```diff
@@ -4,13 +4,21 @@ pkgname=file
 pkgver=5.36
 pkgrel=0
 pkgdesc="File type identification utility"
-url="http://www.darwinsys.com/file"
+url="https://www.darwinsys.com/file"
```
Comment (ncopa) [inline main/file/APKBUILD]: with this change we need bump pkgrel, because `apk info --webpage` will give different result.

Comment (J0WI) [reply main/file/APKBUILD]: Isn't this updated anyway with the next change?

**Comment by ncopa** [comment]
i squashed the commits. thanks!

File: main/file/APKBUILD (hunk 1 above)
Comment (J0WI) [reply main/file/APKBUILD]: btw: would be great if you could give a feedback here:
https://github.com/alpinelinux/aports/pull/4516#issuecomment-429306674

**Pull request closed without merging**